### PR TITLE
Vanilla structure update

### PR DESCRIPTION
**Pull request by RCopeland**
## What does this PR do?
Moves code from spark/ into html/

### Code
 - [x] Unit Testing in Spark Vanilla with `npm run test` (100% coverage, 100% passing)

### CODE REVIEW

File: html/base/inputs/_inputs.scss [hunk 1]
```diff
@@ -7,4 +7,3 @@
 @import './text-icon-input';
 @import './input-container';
 @import './textarea';
-@import './datepicker';
```
Comment (afebbraro) [inline html/base/inputs/_inputs.scss]: Why are we removing datepicker?

File: html/package.json [hunk 2]
```diff
@@ -34,16 +35,29 @@
     "@babel/core": "^7.1.0",
     "@babel/preset-env": "^7.1.0",
     "@babel/register": "^7.0.0",
-    "babel-loader": "^8.0.4",
+    "@storybook/addon-a11y": "^5.2.1",
```
Comment (afebbraro) [inline html/package.json]: We should update the SB package to latest as soon as we can.

File: html/utilities/polyfills/ObjectPrepend.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 // Source: https://github.com/jserz/js_piece/blob/master/DOM/ParentNode/prepend()/prepend().md
 (function (arr) {
-  arr.forEach((item) => {
+  arr.forEach(function (item) {
```
Comment (afebbraro) [inline html/utilities/polyfills/ObjectPrepend.js]: Looks like we are changing the original Spark package here. This needs to be added to release notes as a bug fix.

File: react/package.json [hunk 4]
```diff
@@ -46,6 +46,7 @@
     "rollup-plugin-commonjs": "^10.1.0",
     "rollup-plugin-node-resolve": "^5.2.0",
     "rollup-plugin-replace": "^2.2.0",
+    "sass-loader": "^8.0.0",
```
Comment (afebbraro) [inline react/package.json]: Whats this change for?

File: html/base/inputs/_inputs.scss (hunk 1 above)
Comment (RCopeland) [reply html/base/inputs/_inputs.scss]: matching staging, discussed irl

File: html/utilities/polyfills/ObjectPrepend.js (hunk 3 above)
Comment (RCopeland) [reply html/utilities/polyfills/ObjectPrepend.js]: added

File: react/package.json (hunk 4 above)
Comment (RCopeland) [reply react/package.json]: i missed a dep in the react restructure, discussed irl

**Pull request merged**